benchmarks/startup.py

```diff
@@ -4,7 +4,7 @@
 
 
 class StartupBench(BaseBench):
-    number = 50
+    number = 100
     repeat = 1
```

Comment:
startup does not need new dir each run, hence single statup/teardown and 100 repetitions